src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -0,0 +1,356 @@
+                (new ToggleColumn('active'))
+                    ->setName($this->trans('Status', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'active',
+                        'primary_field' => 'id_product',
+                        'route' => 'admin_products_v2_index',
+                        'route_param_name' => 'productId',
+                    ])
+            )
+            ->add((new ActionColumn('actions'))
+            ->setName($this->trans('Actions', [], 'Admin.Global'))
+            ->setOptions([
+                'actions' => (new RowActionCollection())
+                    ->add((new LinkRowAction('edit'))
+                    ->setName($this->trans('Edit', [], 'Admin.Actions'))
+                    ->setIcon('edit')
+                    ->setOptions([
+                        'route' => 'admin_product_form',
+                        'route_param_name' => 'id',
+                        'route_param_field' => 'id_product',
+                    ])
+                    )
+                    ->add((new SubmitRowAction('delete'))
+                    ->setName($this->trans('Delete', [], 'Admin.Actions'))
+                    ->setIcon('delete')
+                    ->setOptions([
+                        'method' => 'DELETE',
+                        'route' => 'admin_products_v2_delete',
+                        'route_param_name' => 'productId',
+                        'route_param_field' => 'id_product',
+                        'confirm_message' => $this->trans(
+                            'Delete selected item?',
+                            [],
+                            'Admin.Notifications.Warning'
+                        ),
+                    ])
+                    ),
+                //    @todo Implement remaining actions
+                //    ->add((new PreviewProductRowAction('preview_product'))
+                //        ->setName($this->trans('Preview', [], 'Admin.Actions'))
+                //        ->setIcon('remove_red_eye')
+                //        ->setOptions([
+                //            'route' => 'admin_products_preview',
+                //            'route_param_name' => 'productId',
+                //            'route_param_field' => 'id_product',
+                //        ])
+                //    )
+                //    ->add((new SubmitRowAction('duplicate'))
+                //        ->setName($this->trans('Duplicate', [], 'Admin.Actions'))
+                //        ->setIcon('content_copy')
+                //        ->setOptions([
+                //            'route' => 'admin_products_duplicate',
+                //            'route_param_name' => 'productId',
+                //            'route_param_field' => 'id_product',
+                //        ])
+                //    )
+            ])
+            );
+        if ($this->configuration->get('PS_STOCK_MANAGEMENT')) {
+            $columns->addAfter(
```

Comment:
I didn't know we had this `addAfter` It's really nice!! 👍

Reply:
Yeah it's a great thing, sometimes I try to do that when doing something with forms and get disappointed it doesn't exist there :D